jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/annotations/ErrorLocation.kt

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+package software.aws.toolkits.jetbrains.services.cloudformation.annotations
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+data class ErrorLocation @JsonCreator constructor(
```

Comment:
By doing this you have made this your API forever due to you have no version checking.

Are you okay with this?

Reply:
I added version checking, thanks.